src/dataviews/dataview-model-base.js

```diff
@@ -60,8 +60,19 @@ module.exports = Model.extend({
 
     // Retrigger an event when the filter changes
     if (this.filter) {
-      this.listenTo(this.filter, 'change', this._onFilterChanged);
+      this.listenTo(this.filter, 'change', this._reloadMap);
     }
+
+    this.bind('change:operation', this._reloadMap, this);
```

Comment:
I don't think we should leave this here since not all types of dataviews have an `operation` attribute.